packages/altair-app/src/app/modules/altair/utils/index.ts

```diff
@@ -165,6 +165,12 @@ export const getFullUrl = (url: string) => {
     return url;
   }
 
+  // regex to test if given string is an environment variable
+  const regex = /^{{[a-zA-Z._]+}}$/gm;
```

Comment:
There are several variable regexes in [this file](https://github.com/altair-graphql/altair/blob/staging/packages/altair-app/src/app/modules/altair/services/environment/environment.service.ts). Might be better to reuse one of them instead of creating a new one.